Rules Combat.html

```diff
@@ -48,6 +48,11 @@ <h2>Moving</h2>
                     more movement than 1 extra to move through. See the Terrain Guide
                     rules page for specifics.
                 </p>
+                <p class="rules">
+                    You cannot move through spaces occupied by enemies. If you are playing
+                    a team game, then you can move through enemy spaces with no additional
+                    penalty.
```

Comment:
>                                                                                                                   If you are playing
>                     a team game, then you can move through enemy spaces with no additional
>                     penalty.

Should be:
                                                                                                                    If you are playing
                     a team game, then you can move through **_friendly_** spaces with no additional
                     penalty.